Hints.py

```diff
@@ -896,7 +896,9 @@ def buildGanonBossKeyString(world):
             elif world.lacs_condition == 'stones':
                 item_req_string = str(world.lacs_stones) + ' ' + item_req_string
             elif world.lacs_condition == 'dungeons':
-                item_req_string = str(world.lacs_dungeons) + ' ' + item_req_string
+                item_req_string = str(world.lacs_rewards) + ' ' + item_req_string
```

Comment:
Nice catch. I guess bridge has the same issue?

Reply:
I does, but apparently I missed it while being hyper focused on getting the Ganon BK for tokens to work. Fixed that as well.